_beats/dev-tools/ecs-migration.yml

```diff
@@ -15,6 +15,11 @@
 
 # Beat fields
 
+- from: beat.hostname
+  to: host.hostname
+  alias6: true
+  alias: true
```

Comment:
Why are we aliasing this field and not including it in the reindexing script? And shouldn't `beat.hostname` go to `observer.hostname`?

Reply:
Sorry, this is confusing.  This doesn't create an alias, it's just beats' log of their aliases.  The only aliases that are actually created are listed in the description.